docs/dietpi_docs_todo.md

```diff
@@ -50,6 +50,36 @@ Open point, has to be cleared.
 
 - [x] Petru to enable new file structure
 
+### 2.3 Content order
+
+**Proposal:** The content of the HOME area is reordered by exchanging Supported SBCs with Optimised software list and Advanced configuration with Community tutorials.  
+This gives the following benefits:
+
+- "Advanced configuration" is nearer to the system usage description
+- All links to DietPi.com forum ("Community tutorials" and "Troubleshooting" are consecutive)
+
+Then the main order will be:
+
+1. Overview
+2. Installation
+3. Getting started
+4. Supported SBCs
+5. Advanced configuration
+6. Community tutorials
+7. Troubleshooting
+8. ...
+
+#### Decision?
+
+### 2.4 Headings of optimized software
+
+Shall the installation number added to every heading of the installation description?  
+See <http://localhost:8000/software/remote_desktop/#29-tigervnc-server-desktop-for-remote-connection>: Number \[29\] is added as a prefix of the heading.
+
+Or shall it be added as a postfix?
+
+#### Decision?
```

Comment:
Jep that would be very helpful 👍, call if `ID [29]` so it matches with how we call that number elsewhere.

Reply:
Proposal: Decision in telecon.